### PR TITLE
Behat test steps for flash messages, minor Travis and CS updates

### LINKED ISSUE #216: Cover flash messages with functional tests

### PR DESCRIPTION
**Pull request by szymach**
Resolves #216

TODO:

- [x] Create failed form operations scenario

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,6 @@
 composer.lock
 composer.phar
 autoload.php
-bin
\ No newline at end of file
+bin
+features/fixtures/project/web/bundles/*
+features/fixtures/project/app/data.sqlite
```
Comment (norzechowicz) [inline .gitignore]: those lines should be moved to ``features/fixtures/project/.gitignore``

File: Resources/views/base.html.twig [hunk 2]
```diff
@@ -57,14 +57,16 @@
     <div class="row">
         <div class="col-lg-12">
         {% block messages %}
-            {% set typeClass = {success: 'success', warning: 'warning', error: 'danger', info: 'info'} %}
-            {% for type,typeMessages in fsi_admin_messages() %}
-                {% for message in typeMessages %}
-                    <div class="alert alert-{{ typeClass[type] }}">
-                        {{ (message.text)|trans({}, message.domain) }}
-                    </div>
+            <div class="notifications">
```
Comment (norzechowicz) [inline Resources/views/base.html.twig]: ``class="messages"``

**Comment by szymach** [comment]
@fsi-open will this suffice?

**Comment by szymach** [comment]
wait I forgot I made a mistake I need to fix

**Comment by szymach** [comment]
@fsi-open OK I fixed the method for checking if email was changed to an incorrect one, should be fine now

File: features/form/form.feature [hunk 3]
```diff
@@ -49,3 +53,18 @@ Feature: Creating new object
...
+    """
+    Data has been successfully saved.
+    """
+
+  Scenario: Editing an element with invalid data
+    Given there is subscriber with id 1 in database
+    And I am on the "Subscriber Edit" page with id 1
+    When I fill the "Email" field with invalid data
+    And I press form "Save" button
+    Then subscriber with id 1 should have not changed email to invalid one
```
Comment (chives) [inline features/form/form.feature]: `Then subscriber with id 1 should not have its email changed to invalid one` - sounds better

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@fsi-open I was about to change `subscriber with id 1 should not have *his* email changed to invalid one` to `(...) should not have *its* email (...)`, but decided it doesn't matter

**Comment by chives** [comment]
:+1:

**Comment by JarekW** [comment]
:+1:

**Pull request merged**